RULES.md

```diff
@@ -71,7 +71,7 @@ The [AWS Solutions Library](https://aws.amazon.com/solutions/) offers a collecti
 | AwsSolutions-EC23  | The Security Group allows for 0.0.0.0/0 or ::/0 inbound access.                                                                                                                                        | Large port ranges, when open, expose instances to unwanted attacks. More than that, they make traceability of vulnerabilities very difficult. For instance, your web servers may only require 80 and 443 ports to be open, but not all. One of the most common mistakes observed is when all ports for 0.0.0.0/0 range are open in a rush to access the instance. EC2 instances must expose only to those ports enabled on the corresponding security group level.                                                                                                     |
-| AwsSolutions-EC26  | The resource creates one or more EBS volumes that have encryption disabled.                                                                                                                            | With EBS encryption, you aren't required to build, maintain, and secure your own key management infrastructure. EBS encryption uses KMS keys when creating encrypted volumes and snapshots. This helps protect data at rest.                                                                                                                                                                                                                                                                                                                                           |
+| AwsSolutions-EC26  | The resource may create one or more EBS volumes that have encryption disabled.                                                                                                                         | With EBS encryption, you aren't required to build, maintain, and secure your own key management infrastructure. EBS encryption uses KMS keys when creating encrypted volumes and snapshots. This helps protect data at rest.                                                                                                                                                                                                                                                                                                                                           |
```

Comment:
I don't think we need to add `may create`.  In this particular case, even if an EC2 instance overrides unencrypted`blockDeviceMappings`in the Launch Template with encrypted volumes. The Launch Template will still create EC2 instances with unencrypted volumes if used in other places outside of the CDK application.

Reply:
Reverted the changes.

Reply:
I want to note that unless a launch template is actually used by a autoscaling group or an instance inside the CDK application, the launch template is currently not checked by this rule at all to handle this [issue](https://github.com/cdklabs/cdk-nag/issues/1330). However the launch template can still be used outside of CDK.